tools/flang1/flang1exe/dtypeutl.c

```diff
@@ -1417,6 +1424,25 @@ get_type(int n, TY_KIND v1, int v2)
   }
   if (dtype == 0) {
     dtype = STG_NEXT_SIZE(stb.dt, n);
+
+    /* handle the possible !dir$ align XX,
+     *
+     * flang1 reaches here only when creating
+     * new dtype. So flang1 should handle the
+     * possible align pragma for new type here.
+     *
+     * flang1 should store the alignment from flg.x[251]
+     * to DTA, later it will updates the dtype alignment
+     * when in alignment() or lower_put_datatype().
+     *
+     * Note that this pragma should only be used to
+     * this strucutre, so flang1 need to clear the flg.x[251]
```

Comment:
NIT: Please correct capitalization, punctuation, and spelling in the comment. Consider using clang-format to wrap the text to 80 columns for a consistent look.

- `it will updates` &rarr; `it will update`
- `this pragma should only be used to this strucutre` $rarr; `this pragma should apply only to this structure`

Reply:
> NIT: Please correct capitalization, punctuation, and spelling in the comment. Consider using clang-format to wrap the text to 80 columns for a consistent look.

Thank you for your suggestion. I hope I have addressed all of these issues in the latest patches.

> 
>     * `it will updates` → `it will update`
> 
>     * `this pragma should only be used to this strucutre` → `this pragma should apply only to this structure`